### ISSUE TITLE
Community 24.1.3 crash PostgreSQL colors inside Dollar-Quoted String Constants ($$..$$)

### ISSUE THREAD

**Issue by shtoormanSam** [dbeaver/dbeaver#35015]
### Description

new Community 24.1.3 colors the code inside PostgreSQL $$-span like text literal. 
It seems "- Specific functions were supported in the SQL Editor (thanks to @EastLord)"

How to disable this future? 
I have tonns of code like
```
create or replace function f()
 returns record
 language plpgsql
as $function$
declare
...
begin
...
end;
$function$
```
and this 'future' makes it hard to read.

I’m grateful for your efforts and your support!

### DBeaver Version

Community 24.1.3

### Operating System

Win10

### Steps to reproduce

just compare colorization in Community 24.1.3 and 24.1.2  for code like this:

```
create or replace function f()
 returns record
 language plpgsql
as $function$
declare
...
begin
...
end;
$function$
```

**Comment by Matvey16** [comment]
@shtoormanSam Can you please show me how this preference page looks like for your connection. And if possible give me a ddl of any function that is colored incorrectly

![image](https://github.com/user-attachments/assets/6787d740-d5fb-4de4-bceb-2ac8c6fdabef)

**Comment by shtoormanSam** [comment]
Thanks, your image help solve the problem!

пн, 29 июл. 2024 г. в 20:00, Matvey16 ***@***.***>:

**Comment by shtoormanSam** [comment]
Hi!
Sorry,  I hurried to report a successful solution - it is also actual for
the latest 21.1.4 version: [image: DBeaver double-quoted-bug.jpg]

сб, 3 авг. 2024 г. в 09:31, Владимир Матвеев ***@***.***>:

**Comment by shtoormanSam** [comment]
in addition, screen-cast


ср, 7 авг. 2024 г. в 10:14, Владимир Матвеев ***@***.***>:

**Comment by E1izabeth** [comment]
@shtoormanSam unfortunately, the attachments that you send with email don't come to the github issue and we don't wee it. The email you get is a notification from github platform, not from us.

**Comment by E1izabeth** [comment]
As a workaround, you can turn the setting `Enable semantic analysis` in `Preferences->Editors->SQL Editor->Code Editor`

![image](https://github.com/user-attachments/assets/02b9de1b-7663-465f-b97d-6665ba37b84e)

**Comment by shtoormanSam** [comment]
Thanks! That was helpful!

ср, 14 авг. 2024 г. в 11:20, Elizabeth ***@***.***>:

**Comment by E1izabeth** [comment]
The issue was fixed in #34972
